frontend/src/components/supp-allowances/TransportationAllowance.vue

```diff
@@ -84,7 +84,7 @@
               <p>Vehicle financing/Lease cost per month: (If any)</p>
             </v-col>
             <v-col cols="6" xl="7" align="center" class="px-2">
-              <AppNumberInput v-model.lazy="model.monthlyLease" :format="monthlyLeaseFormat" required :disabled="readonly" prefix="$" maxlength="12" :rules="[rules.max(5000000)]"></AppNumberInput>
+              <AppNumberInput v-model.lazy="model.monthlyLease" :format="monthlyLeaseFormat" required :disabled="readonly" prefix="$" maxlength="7" :rules="[rules.max(5000000)]"></AppNumberInput>
```

Comment:
Hi Jen, do we have any requirement on the max value of this Vehicle Financing/Lease field? Also, if maxlength is 7, I don't think we need rules.max(5,000,000).

Reply:
agreed and done